services/workflows-service/src/transaction/transaction.module.ts

```diff
@@ -7,18 +7,11 @@ import { TransactionControllerExternal } from '@/transaction/transaction.control
 import { PrismaModule } from '@/prisma/prisma.module';
 import { ProjectScopeService } from '@/project/project-scope.service';
 import { SentryService } from '@/sentry/sentry.service';
-import { TransactionFactory } from '@/transaction/test-utils/transaction-factory';
 
 @Module({
   imports: [ACLModule, PrismaModule],
   controllers: [TransactionControllerInternal, TransactionControllerExternal],
-  providers: [
-    TransactionService,
-    TransactionRepository,
-    ProjectScopeService,
-    SentryService,
-    TransactionFactory,
-  ],
-  exports: [ACLModule, TransactionService, TransactionFactory],
+  providers: [TransactionService, TransactionRepository, ProjectScopeService, SentryService],
+  exports: [ACLModule, TransactionService],
```

Comment:
Main change is removing TransactionFactorty. It is not in use and it has an invalid constructor for dependency injection as its using an object over positional arguments. Also it does not use dependency injection apart of the prisma service which again did not work until now.